### DISCUSSION TITLE
[NMS] Move from MariaDB to MySQL for Bare Metal Deployment

### DISCUSSION THREAD

**Discussion by ShubhamTatvamasi** [magma/magma#5244] [General]
When I deployed NMS on my Bare Metal instance, I faced a lot of issues regarding NMS crash and that was due to MariaDB.
After doing a lot of debugging when I switched to MySQL everything started working perfectly.
We should move to MySQL same as our Minikube deployment.

[Bare Metal MariaDB helm install](https://github.com/magma/magma/blob/v1.4/orc8r/cloud/deploy/bare-metal/deploy_charts.sh#L65)

[Minikube MySQL helm install](https://github.com/magma/magma/blob/v1.4/docs/readmes/orc8r/development/minikube_deployment.md#setup-minikube-and-helm)

**Comment by mattymo** [comment]
@ShubhamTatvamasi It would be easier to debug your issue with MariaDB and NMS crashing than to switch to MySQL. The open source community embraces MariaDB much more for a variety of reasons that I won't get into.

Regarding Magma as a whole, there is a push to move to postgresql actually. NMS doesn't support it correctly now, but it should be fixed relatively soon. It works in HA much more natively and seamlessly on Kubernetes.

**Reply by ShubhamTatvamasi** [reply]
So in the future NMS will be moved to PostgreSQL?

**Reply by hcgatewood** [reply]
We're in the process of moving NMS to Postgres, so that we can use a single DB instance across NMS and Orc8r

However, we have existing support for Maria in both NMS and Orc8r, and the current plan is to retain that support
